operators/annotationlab/4.4.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,14 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: annotationlab
+  operators.operatorframework.io.bundle.channels.v1: alpha
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.25.0
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  operators.operatorframework.io.metrics.project_layout: helm.sdk.operatorframework.io/v1
+
```

Comment:
Please add openshift version range, because your operator uses deprecated api which is not present in ocp v4.12 or k8s v1.25
```
com.redhat.openshift.versions: v4.8-v4.11
```

Reply:
Please also fix other versions.
```
v3.4.0
v4.1.2
v4.2.0
v4.2.1
v4.3.0
v4.4.0
```

This change can be done in this PR